packages/core/src/api.ts

```diff
@@ -17,29 +17,54 @@ export class API {
     return this._dsnObject;
   }
 
-  /** Returns a string with auth headers in the url to the store endpoint. */
```

Comment:
This was the documentation of `getStoreEndpoint`! Fixed, it **does not** include auth.